a11y.js

```diff
@@ -9,21 +9,20 @@ const services = require('./cypress/support/config/services');
 const pageWidths = [360];
 const baseUrl = 'http://localhost:7080';
 
-const getPageTypes = (service) =>
-  pathOr(null, [service, 'pageTypes'], services);
+const getPageTypes = service => pathOr(null, [service, 'pageTypes'], services);
 
-const getSmokePaths = (config) => {
+const getSmokePaths = config => {
   const { environments, smoke } = config;
   return smoke && environments ? environments.local.paths : null;
 };
 
-const getUrls = (pageType) =>
+const getUrls = pageType =>
   Object.keys(services)
     .map(getPageTypes)
-    .map((pageTypes) => pathOr(null, [pageType], pageTypes))
+    .map(pageTypes => pathOr(null, [pageType], pageTypes))
     .map(getSmokePaths)
-    .filter((page) => !!page)
-    .map((url) => `${baseUrl}${url}`);
+    .filter(page => !!page)
```

Comment:
```suggestion
    .filter(Boolean)
```

Reply:
I think I can actually remove it altogether.

Reply:
Mmm nope, no idea why but will use `filter(Boolean)` instead